### PR TITLE
 Adding SAIPTF tests for L3IPv4_32Test

### PR DESCRIPTION
**Pull request by chitra-raghavan**
Adding SAI PTF test for L3IPv4_32Test

(https://github.com/opencomputeproject/SAI/blob/master/test/saithrift/tests/SAITest.md)

### CODE REVIEW

File: test/saithrift/tests/sail3.py [hunk 1]
```diff
@@ -3450,3 +3450,130 @@ def verifyEcmpPktDistributionIpv6(self,max_itrs,ports,member):
...
+        rif_id3 = sai_thrift_create_router_interface(self.client, vr_id, SAI_ROUTER_INTERFACE_TYPE_PORT, port3, 0, v4_enabled, v6_enabled, mac)
+
+        ### create neighbor and next hop
+        sai_thrift_create_neighbor(self.client, addr_family, rif_id2, ip_addr2, dmac2)
+        nhop1 = sai_thrift_create_nhop(self.client, addr_family, ip_addr2, rif_id2)
+
+        sai_thrift_create_neighbor(self.client, addr_family, rif_id1, ip_addr1, dmac1)
+
+        ### create route
+        sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr1_subnet, ip_mask1, nhop1)
```
Comment (MELLANOX-zhichenr) [inline test/saithrift/tests/sail3.py]: Test is passing after adding local route 20.20.20.0/24 to rif_id2, and local route 10.1.1.0/24 to rif_id1

Comment (chitra-raghavan) [reply test/saithrift/tests/sail3.py]: Addressed review comments

**Comment by chitra-raghavan** [comment]
> need to add local routes suggested by zhichen so that test can pass.

Added local routes

File: test/saithrift/tests/sail3.py [hunk 2]
```diff
@@ -3514,6 +3517,9 @@ def runTest(self):
 
         ### create route
         sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr1_subnet, ip_mask1, nhop1)
+        sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr_subnet, ip_mask, rif_id1 )
```
Comment (MELLANOX-zhichenr) [inline test/saithrift/tests/sail3.py]: Thanks.
Please remove the redundant space at the end of line 3520.
Please also remove the local routes in cleanup stage.

Comment (chitra-raghavan) [reply test/saithrift/tests/sail3.py]: updated cleanup and remove the redundant space

**Pull request merged**